controllers/storagecluster/reconcile.go

```diff
@@ -324,6 +327,11 @@ func (r *StorageClusterReconciler) reconcilePhases(
 			if err := r.deleteResources(instance); err != nil {
 				r.Log.Info("Uninstall in progress.", "Status", err)
 				r.recorder.ReportIfNotPresent(instance, corev1.EventTypeWarning, statusutil.EventReasonUninstallPending, err.Error())
+
+				if !isErrorResolvable[instance.UID] {
```

Comment:
This change is not related to OCS to OCS. Should it be in a separate PR?

Reply:
But OCS to OCS is the one that requires this functionality and I think creating a separate PR will delay things.